.github/workflows/doc-build.yml

```diff
@@ -25,10 +25,6 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Update PATH for west
-      run: |
-        echo "$HOME/.local/bin" >> $GITHUB_PATH
```

Comment:
I guess recent ubuntu is doing this for the user automatically? It was required on that OS at some point in the past.

Reply:
This is the `PATH` of the `ubuntu-latest` runner:

```
PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:/home/runner/.local/bin:/opt/pipx_bin:/usr/share/rust/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
```

so yes, `/home/runner/.local/bin` is already there.